modules/life-cycle-dates.adoc

```diff
@@ -9,10 +9,10 @@
 [options="header"]
 |===
 |Version    |General availability   |End of life
-|4.13       |May 17, 2023           |Jul 17, 2024
-|4.12       |Jan 17, 2023           |Mar 17, 2024
-|4.11       |Aug 10, 2022           |Oct 10, 2023
-|4.10       |Mar 10, 2022           |Jul 10, 2023
+|4.13       |May 17, 2023           |Sep 17, 2024
+|4.12       |Jan 17, 2023           |May 17, 2024
+|4.11       |Aug 10, 2022           |Dec 10, 2023
+|4.10       |Mar 10, 2022           |Sep 10, 2023
```

Comment:
September 10, 2023 is 18 months from March 10, 2022. Is this the correct end of the 4.10 lifecycle?

Also, a major support change such as this needs to go through our docs change management process. cc: @kalexand-rh 

I recognize that there is some urgency in getting this change to customers, but it is important that we loop in support. Please let me know if you need any help going through the change management process.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

Reply:
+1 to sending this through PM so that it's more visible to internal stakeholders. I agree that it's particularly important for support to be aware of this one and that the 4.10 EOL date needs to be confirmed. If it's mean to be 18 months, that's fine, but we need to make sure that we're not _calling_ it 16 months in the docs.

Reply:
Hello @michaelryanpeter,
   I did some digging regarding the discrepancy of those dates (good catch) and that indeed is the correct date. Will make sure that it is not called out as a 16 month extension in ROSA/OSD docs. 
https://redhat-internal.slack.com/archives/C021EMWAQ76/p1685711842357079
@kalexand-rh

Reply:
Thank you for looking into that!

Reply:
> Thank you for looking into that!

You are welcome!